packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -110,7 +119,16 @@ class ChartController extends DisposableController
     _xAxisLabeledTimestamps[rightLabelIndex] = timestamp;
   }
 
+  /// If false displays top horizontal line of chart.
```

Comment:
I assume you mean hides instead of displays.
Normalize these comments to all be in terms of Display instead of some in terms of display and some in terms of hide. I think the one for displayXLabels is the clearest.

Reply:
Yes it should be. Done.